packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -553,12 +552,7 @@ export function getDeviceSpec() {
  */
 export class ContainerRuntime
 	extends TypedEventEmitter<IContainerRuntimeEvents>
-	implements
-		IContainerRuntime,
-		IGarbageCollectionRuntime,
-		IRuntime,
-		ISummarizerRuntime,
-		ISummarizerInternalsProvider
+	implements IContainerRuntime, IRuntime, ISummarizerRuntime, ISummarizerInternalsProvider
```

Comment:
`ISummarizerRuntime` and `ISummarizerInternalsProvider` are also similar to `IGarbageCollectionRuntime` in the sense that they are used internally only. Why just remove `IGarbageCollectionRuntime`?

Reply:
I didn't notice them :P

Reply:
Looked at it, they'll have the same problem - they're constructor params for Summarizer class which is exported. So leaving them.